modules/navier_stokes/test/tests/finite_volume/ins/wall_distance_capped_mixing_length_aux/capped_mixing_length.i

```diff
@@ -3,7 +3,7 @@ von_karman_const = 0.41
 H = 1 #halfwidth of the channel
 L = 150
 
-Re = 13700
+Re = 100
```

Comment:
This is the test that doesn't converge well after the change to SSF because SSF now enforces that the incoming flux is what the user thinks it will be. If we don't reduce `Re` here, another option is turning this into a transient test. Probably would need to make it into a heavy test if we want to run it all the way to stead-state

Reply:
ah ok. Transients is a good idea imo.

Reply:
Did you try modifying the mesh/mixing length parameters?

Reply:
No. Given that you guys don't like the change to `Re`, I think just running this as a transient to steady-state is my favorite option as it shows that we still have no problem with solving for a high `Re`; it's just that the change to use the user-provided inlet conditions exactly pushes this problem outside of Newton's ball-of-convergence for a trivial IC and `Steady` solve